sae/sae.py

```diff
@@ -65,23 +65,33 @@ def __init__(
     @staticmethod
     def load_many_from_hub(
         name: str,
+        local: bool = False,
```

Comment:
If the method is called `load_many_from_hub`, having a `local=True` option is pretty weird. I would prefer to either not support `local` or rename to just `load_many`

Reply:
agreed!